packages/snaps-sdk/src/jsx/components/Icon.ts

```diff
@@ -0,0 +1,43 @@
+import { createSnapComponent } from '../component';
+
+/**
+ * The props of the {@link Icon} component.
+ *
+ * @property name - The name of the icon to display from a pre-defined list.
+ * @property color - The color of the displayed icon.
+ * @property size - The size of the displayed icon. Use `inherit` to size it the same as the text.
+ */
+export type IconProps = {
+  name: string;
+  color?:
+    | 'default'
+    | 'primary'
+    | 'error'
+    | 'success'
+    | 'warning'
+    | 'info'
+    | undefined;
+  size?: 'xs' | 'sm' | 'md' | 'lg' | 'xl' | 'inherit' | undefined;
```

Comment:
Wondering if we should just support one size for now and add the flexibility later.

Generally we've been slow and cautious when adding additional customization to the components.

Reply:
So what's the verdict on your side?

Reply:
I think I would just omit `size` for now

Reply:
I've followed @eriknson suggestion below